docs/learn-faq.md

```diff
@@ -6,80 +6,327 @@ sidebar_label: Frequently Asked Questions (FAQs)
 
 _This FAQ focuses on technical questions for users interested in developing applications for Polkadot. If you have a more general question, you may wish to search for the answer on the main [Polkadot Network FAQ](https://polkadot.network/faq). If you have a question which is not answered, please feel free to ask on the Polkadot Watercooler [Riot channel](https://riot.im/app/#/room/#polkadot-watercooler:matrix.org)._
 
-### How many validators will be validating at Mainnet launch?
-
-Most likely somewhere between 50 - 100, increasing to 1000 or so later on. [Source](https://youtu.be/IRc5Jma_eH8?t=1630) The number of validators can be changed by governance, and so it is difficult to state precisely how many will be in the active validator set at any particular point in the future.
+## Polkadot Launch
+
+### What is the launch process of Polkadot Beta?
+
+The Polkadot network will launch first in as a Proof-of-Authority (PoA) chain.
+During this time, governance will be controlled by the single Sudo (super-user)
+account, and functionality will be restricted to (among other things) disallow
```

Comment:
"restricted to disallow" sounds strange (as it can be interpreted as restricting the disallowance, depending on how one reads double negatives).  How about... "...and functionality will be restricted.  Among other things, transfer of funds will not be allowed during this period."